pkg/wfc3/calwf3/wf3rej/rej_command.c

```diff
@@ -15,6 +15,36 @@ static int getArgS (char **, int, int *, short *);
 static int getArgR (char **, int, int *, float *);
 static int getArgT (char **, int, int *, char *);
 
+static void printSyntax(void) {
+    printf("    syntax:  wf3rej.e input output [-t] [-v]\n");
+    printf("                      [-shadcorr] [-crmask]\n");
+    printf("                      [-table <filename>] [-scale #]\n");
+    printf("                      [-init (med|min)] [-sky (none|mode)]\n");
+    printf("                      [-sigmas #] [-radius #] [-thresh #]\n");
+    printf("                      [-pdq #]\n");
+    printf("                      [-r]\n");
```

Comment:
I can't seem to find any explanation for the `-r` argument.

Reply:
Oops.  Fixed in dc347ca.